src/twisted/web/test/test_cgi.py

```diff
@@ -201,13 +209,14 @@ def checkResponse(response):
 
     def test_duplicateHeaderCGI(self):
         """
-        If a CGI script emits two instances of the same header, both are sent in
-        the response.
+        If a CGI script emits two instances of the same header, both
```

Comment:
Minor nitpick: Previously, the line was exactly 80 characters. According to the coding standard, lines are flowed at 79, so this it is correct to reformat this.  However, _this_ line is 68 characters, which is 10 too _short_, suggesting that your editor is misconfigured.